### PR TITLE
DVCSMP-1699 Na02>Missing/Failed Oauth Tokens - Life360

### PR DESCRIPTION
**Pull request by rohandesai**
- now calling the method to get the right shard url for Life360

@llinder @jasonbio

### CODE REVIEW

File: smartapps/smartthings/life360-connect.src/life360-connect.groovy [hunk 1]
```diff
@@ -192,7 +192,7 @@ def getSmartThingsClientId() {
    return "pREqugabRetre4EstetherufrePumamExucrEHuc"
 }
 
-def getServerUrl() { appSettings.serverUrl }
```
Comment (llinder) [inline smartapps/smartthings/life360-connect.src/life360-connect.groovy:195]: should we remove the old serverUrl from the smartapp settings?

**Comment by llinder** [comment]
Seems fine as long as Life360 doesn't start validating redirect URLs in their OAuth flow.  If they ever do this will break and we will need to make further updates.

**Comment by rohandesai** [comment]
@llinder addressed comment for this too.

**Comment by jasonbio** [comment]
👍

**Pull request merged**